services/svc/schemaregistry/Cargo.toml

```diff
@@ -21,7 +21,7 @@ azure_identity = { path = "../../../sdk/identity", version = "0.1" }
 tokio = { version = "1.0", features = ["macros"] }
 
 [package.metadata.docs.rs]
-all-features = true
+features = []
```

Comment:
Doesn't this mean nothing gets documented? I'd expect this to be a subset of `features.default` <= 5 features. This might be a symptom of no default with which I was concerned above.

Reply:
The `default` feature gets documented.

Reply:
This can be tested with:
```
cargo doc -p azure_svc_schemaregistry --no-deps --open --features "no-default-version"
```
![image](https://user-images.githubusercontent.com/80104/156875231-9f048762-7660-4695-84fc-7744ee444028.png)